### PR TITLE
For #5359 , #5360 - Update ETP Vectors for Android 5+6

### PR DESCRIPTION
**Pull request by ekager**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/trackingprotection/SwitchWithDescription.kt [hunk 1]
```diff
@@ -28,19 +29,19 @@ class SwitchWithDescription @JvmOverloads constructor(
                 R.styleable.SwitchWithDescription_switchIcon,
                 R.drawable.ic_tracking_protection
             )
-            switch_widget?.setCompoundDrawablesWithIntrinsicBounds(
-                resources.getDrawable(
-                    id,
-                    context.theme
+            switch_widget.setCompoundDrawablesWithIntrinsicBounds(
```
Comment (NotWoods) [inline app/src/main/java/org/mozilla/fenix/trackingprotection/SwitchWithDescription.kt]: This is unrelated but it'd be nice to use the `putCompoundDrawablesWithIntrinsicBounds` method so we can use named parameters here.

File: app/src/main/res/xml/preferences.xml [hunk 2]
```diff
@@ -55,7 +55,7 @@
         android:title="@string/preferences_category_privacy"
         app:iconSpaceReserved="false">
         <androidx.preference.Preference
-            android:icon="@drawable/ic_tracking_protection_enabled"
+            app:icon="@drawable/ic_tracking_protection_enabled"
```
Comment (NotWoods) [inline app/src/main/res/xml/preferences.xml]: I think preferences.xml should automatically use compat with no changes. Does the settings screen still show the icon?

File: app/src/main/res/xml/tracking_protection_preferences.xml [hunk 3]
```diff
@@ -12,7 +12,7 @@
         app:allowDividerBelow="false" />
     <SwitchPreference
         android:defaultValue="true"
-        android:icon="@drawable/ic_tracking_protection_enabled"
+        app:icon="@drawable/ic_tracking_protection_enabled"
```
Comment (NotWoods) [inline app/src/main/res/xml/tracking_protection_preferences.xml]: ditto

File: app/src/main/res/xml/preferences.xml (hunk 2 above)
Comment (ekager) [reply app/src/main/res/xml/preferences.xml]: Yeah u right I can't reproduce idk why I thought this was broken too 🤷‍♀

File: app/src/main/java/org/mozilla/fenix/trackingprotection/SwitchWithDescription.kt (hunk 1 above)
Comment (ekager) [reply app/src/main/java/org/mozilla/fenix/trackingprotection/SwitchWithDescription.kt]: Updated!

**Pull request merged**